cli/src/main/java/org/jboss/sbomer/cli/feature/sbom/processor/WorkaroundMissingNpmDependencies.java

```diff
@@ -74,11 +74,8 @@ public void analyzeBuild(Component component, Build build) {
             return;
         }
         String buildId = build.getId();
-        List<Artifact> npmDependencies = buildsWithNpmDependencies.get(buildId);
-        if (npmDependencies == null) {
-            npmDependencies = new ArrayList<>(pncService.getNPMDependencies(buildId));
-            buildsWithNpmDependencies.put(buildId, npmDependencies);
-        }
+        List<Artifact> npmDependencies = buildsWithNpmDependencies
+                .computeIfAbsent(buildId, k -> new ArrayList<>(pncService.getNPMDependencies(k)));
```

Comment:
Note that this is an actual code change:

Replace this "Map.get()" and condition with a call to "Map.computeIfAbsent()".